src/terminal/adapter/adaptDispatch.cpp

```diff
@@ -3294,10 +3294,6 @@ bool AdaptDispatch::_EraseAll()
     // Also reset the line rendition for the erased rows.
     textBuffer.ResetLineRenditionRange(newViewportTop, newViewportBottom);
 
-    // Clear any marks that remain below the start of the
-    textBuffer.ClearMarksInRange(til::point{ 0, newViewportTop },
```

Comment:
why not? this code doesn't call ClearScrollback, shouldn't it destroy the marks it cleared?